diff --git a/config.def.h b/config.def.h
index d805331..0d63a68 100644
--- a/config.def.h
+++ b/config.def.h
@@ -63,7 +63,27 @@ static const char unknown_str[] = "n/a";
  * wifi_essid          WiFi ESSID                      interface name (wlan0)
  * wifi_perc           WiFi signal in percent          interface name (wlan0)
  */
+
+/* TODO: Dynamically select the correct ifaces, depending on hostname. */
+/* TODO: Add a module to show VPN state. */
+/* LAPTOP */
+static const char bname[] = "BAT0";
+static const char iname_eth[] = "enp1s0";
+/* PC */
+static const char iname_eth[] = "enp4s0";
+
+/* Date format https://cplusplus.com/reference/ctime/strftime/ */
+
 static const struct arg args[] = {
 	/* function format          argument */
-	{ datetime, "%s",           "%F %T" },
+	{ battery_state,  "%s", bname },
+	{ battery_perc,   "b:%3s", bname },
+	{ run_command,    " | ♫:%3s", "/bin/bash -c \"amixer get Master | awk '\\$0~/%/{print \\$5}' | tr -d '[]%'\""},
+	{ netspeed_tx,    " | W: ▲%9s", iname_wifi},
+	{ netspeed_rx,    " ▼%9s", iname_wifi},
+	{ netspeed_tx,    " | E: ▲%9s", iname_eth},
+	{ netspeed_rx,    " ▼%9s", iname_eth},
+	{ ram_perc,       " | R:%3s", NULL},
+	{ cpu_perc,       " | C:%3s", NULL},
+	{ datetime, "%s", " | %a %d %b %T" },
 };
diff --git a/slstatus.c b/slstatus.c
index cb54f29..b2521a7 100644
--- a/slstatus.c
+++ b/slstatus.c
@@ -86,7 +86,7 @@ main(int argc, char *argv[])
 		status[0] = '\0';
 		for (i = len = 0; i < LEN(args); i++) {
 			if (!(res = args[i].func(args[i].args)))
-				res = unknown_str;
+				continue;
 
 			if ((ret = esnprintf(status + len, sizeof(status) - len,
 			                     args[i].fmt, res)) < 0)
